packages/ebsr/configure/src/main.jsx

```diff
@@ -26,12 +26,41 @@ export class Main extends React.Component {
     onConfigurationChanged: PropTypes.func
   };
 
+  onModelChanged = (model, key) => {
+    const { onModelChanged } = this.props;
+
+    if (key !== 'partB.choiceMode') {
+      return onModelChanged(model);
+    };
+
+    let value = model.partB.choiceMode;
```

Comment:
"value" can be const (it doesn't get updated)

Reply:
sure